Makefile

```diff
@@ -191,8 +191,17 @@ docker-integration-test:
 	@./scripts/integration-tests/docker-integration-test.sh
 	@cd scripts/integration-tests/prometheus/ && ./prometheus-integration-test.sh
 
-.PHONY: metalint
-metalint: $(patsubst %,metalint-%,$(SUBDIRS))
+SUBDIR_TARGETS = mock-gen thrift-gen proto-gen asset-gen all-gen metalint test \
+				 test-xml
```

Comment:
Can you add `test-ci-unit` and `test-ci-integration` to this list?

Reply:
Sure thing. Guess I missed those.